coderd/database/queries/notifications.sql

```diff
@@ -189,3 +189,28 @@ WHERE
 INSERT INTO notification_report_generator_logs (notification_template_id, last_generated_at) VALUES (@notification_template_id, @last_generated_at)
 ON CONFLICT (notification_template_id) DO UPDATE set last_generated_at = EXCLUDED.last_generated_at
 WHERE notification_report_generator_logs.notification_template_id = EXCLUDED.notification_template_id;
+
+-- name: GetNotificationPushSubscriptionsByUserID :many
+SELECT *
+FROM notification_push_subscriptions
+WHERE user_id = @user_id::uuid;
+
+-- name: InsertNotificationPushSubscription :one
+INSERT INTO notification_push_subscriptions (id, user_id, created_at, endpoint, endpoint_p256dh_key, endpoint_auth_key)
```

Comment:
do not allow supplying id because the table specifies id as default gen_random_uuid